projects/tinkerbell/sandbox/docker-compose.yml

```diff
@@ -198,41 +131,25 @@ services:
       PACKET_VERSION: ${PACKET_VERSION:-ignored}
       ROLLBAR_TOKEN: ${ROLLBAR_TOKEN:-ignored}
       ROLLBAR_DISABLE: ${ROLLBAR_DISABLE:-1}
-      MIRROR_HOST: ${TINKERBELL_HOST_IP:-127.0.0.1}:8080
-      OSIE_PATH_OVERRIDE: https://projectbuildpipeline-857-pipelineoutputartifactsb-10ajmk30khe3f.s3.us-west-2.amazonaws.com/projects/tinkerbell/hook/latest/54bae38625828b69259eccb3357e13c269437e28 
+      OSIE_PATH_OVERRIDE: https://tinkerbell-storage-for-eksa.s3.us-west-2.amazonaws.com/misc/osie/current
       DNS_SERVERS: 8.8.8.8
       PUBLIC_IP: $TINKERBELL_HOST_IP
       BOOTP_BIND: $TINKERBELL_HOST_IP:67
       HTTP_BIND: $TINKERBELL_HOST_IP:80
       SYSLOG_BIND: $TINKERBELL_HOST_IP:514
       TFTP_BIND: $TINKERBELL_HOST_IP:69
-      DOCKER_REGISTRY: $TINKERBELL_HOST_IP 
-      REGISTRY_USERNAME: ${TINKERBELL_REGISTRY_USERNAME:-admin}
-      REGISTRY_PASSWORD: ${TINKERBELL_REGISTRY_PASSWORD:-Admin1234}
+      DOCKER_REGISTRY: public.ecr.aws/l0g8r8j6
```

Comment:
shouldn't the registry just be `public.ecr.aws`?

Reply:
sure, that's fine too. Then we need to make sure your paths for images are correct. I wasn't sure if we had decided exactly what to put here